traefik/README.md

```diff
@@ -26,55 +22,112 @@ For Agent v7.21+ / v6.21+, follow the instructions below to install the Traefik
 
 ### Configuration
 
-1. Edit the `traefik.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your Traefik [metrics](#metric-collection) or [logs](#log-collection). See the [sample traefik.d/conf.yaml][8] for all available configuration options.
+<!-- xxx tabs xxx -->
+<!-- xxx tab "v2" xxx -->
 
-2. [Restart the Agent][9].
+#### About v2
+For information about the changes from v1 to v2, see the [Traefik migration guide][5]. For information about the latest version, see [Traefik documentation][6].
 
 #### Metric collection
 
-Add this configuration setup to your `traefik.yaml` file to start gathering your [metrics][10]:
+Follow [Traefix's documentation][7] to send [Traefix metrics][8] to Datadog.
+
+#### Log collection
+
+**Available for Agent >6.0**
+
+By default, [Traefik logs][9] are sent to stdout. This should not be changed for containerized version, as the Datadog Agent is able to collect logs directly from container `stdout`/`stderr`.
```

Comment:
```suggestion
By default, [Traefik logs][9] are sent to stdout. This should not be changed for containerized version, because the Datadog Agent can collect logs directly from the container `stdout`/`stderr`.
```